analyses/chromosomal-instability/01-plot-chromosomal-instability.Rmd

```diff
@@ -88,7 +90,32 @@ breaks_list <- readr::read_rds(file.path("breakpoint-data", "breaks_lists.RDS"))
 Get a vector of the biospecimen IDs. 
 
 ```{r}
-common_samples <- unique(breaks_list$union_of_breaks$samples)
+common_samples <- unique(breaks_list$intersection_of_breaks$samples)
+```
+
+Set up uncallable regions BED. 
+
+```{r}
+uncallable_bed <- readr::read_tsv(
+  file.path(
+    "..",
+    "copy_number_consensus_call",
+    "ref",
+    "cnv_excluded_regions.bed"
+  ),
+  col_names = c("chrom", "start", "end")
+) %>%
+  # Reformat the chromosome variable to drop the "chr"
+  dplyr::mutate(chrom = factor(gsub("chr", "", chrom),
+    levels = c(1:22, "X", "Y")
+  )) %>%
+  dplyr::filter(
+    !is.na(chrom),
+    # Exclude the small regions
+    end - start > 200000,
```

Comment:
Do we want to exclude the small regions if we are going to calculate overlaps using them later?

Reply:
We can keep them.